deploy/009-configure-vaults.ts

```diff
@@ -0,0 +1,67 @@
+import deployedContracts from "@venusprotocol/governance-contracts/deployments/deployments.json";
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+interface AdminAccounts {
+  [key: string]: string;
+}
+const adminAccount: AdminAccounts = {
+  sepolia: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb", // SEPOLIA MULTISIG
+  ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67", // ETHEREUM MULTISIG
+};
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts } = hre;
+  const { deployer } = await getNamedAccounts();
+
+  const chainId = (await hre.getChainId()) as keyof typeof deployedContracts;
+  const accessControlManager = hre.network.live
+    ? await ethers.getContractAt(
+        "AccessControlManager",
+        deployedContracts[chainId][0].contracts.AccessControlManager.address,
+      )
+    : await ethers.getContract("AccessControlManager");
+
+  const xvs = await ethers.getContract("XVS");
+  const xvsVaultProxyDeployment = await ethers.getContract("XVSVaultProxy");
+  const xvsStoreDeployment = await ethers.getContract("XVSStore");
+
+  const xvsVaultProxy = await ethers.getContractAt("XVSVault", xvsVaultProxyDeployment.address);
+
+  let txn = await xvsVaultProxy.setXvsStore(xvs.address, xvsStoreDeployment.address);
+  await txn.wait();
+
+  txn = await xvsVaultProxy.setAccessControl(accessControlManager.address);
+  await txn.wait();
+
+  if (!hre.network.live) {
```

Comment:
`bsctestnet`, in hardhat.config.ts, doesn't have `live: true`. Should we set it?